service/src/main/java/bio/terra/workspace/db/DbUtils.java

```diff
@@ -22,7 +22,11 @@ public static String setColumnsClause(MapSqlParameterSource columnParams) {
       if (i > 0) {
         sb.append(", ");
       }
-      sb.append(columnName).append(" = :").append(columnName);
+      if (!columnName.equals("attributes")) {
```

Comment:
Having the utility method have knowledge of specific column names seems fragile.
How about making a call form like
```
setColumnClause(MapSqlParameterSource columnParams, String... jsonColumns)
```
So the caller determines the column names to cast?

Reply:
not sure if i did what you mean. Could you take a look at the new implementation?